### ISSUE TITLE
Implement Classification with Softmax activation function/Logistic in tensorflow and pytorch

### ISSUE THREAD

**Issue by yap231995** [Gurobi/gurobi-machinelearning#347]
I would like to have to have softmax activation function as my variables each probability of the output of the neural network. 
This will help very much, instead of just regression.

**Comment by pobonomo** [comment]
Hi,
This is something that I have been playing with as Gurobi is improving it's capacity to solve nonlinear problems.
A model is not difficult to do but note that the size of the networks that we currently can solve is (in my limited tries) very small.

Do you have a case where it would be useful with a very small network?

**Comment by pobonomo** [comment]
The PR #473 would actually implement this. Will try to find some nice example to make it worth the merge.
